RestaurantReservation.Db/DataModels/ReservationView.cs

```diff
@@ -0,0 +1,11 @@
+﻿namespace RestaurantReservation.Db.DataModels
+{
+    public class ReservationView
+    {
+        public int ReservationId { get; set; }
+        public string FirstName { get; set; }
```

Comment:
It's not clear , please rename to CustomerFirstName , 
other contributers may be confused think it's employee's name 
please add more details for customer and restaurant information